packages/rpc-core/src/rpc-methods/getInflationGovernor.ts

```diff
@@ -0,0 +1,30 @@
+import { Commitment } from './common';
+
+type GetInflationGovernorApiResponse = Readonly<{
+    /** The initial inflation percentage from time 0 */
+    initial: number; // Until we land on best type for `f64`
+    /** Percentage of total inflation allocated to the foundation */
+    foundation: number; // Until we land on best type for `f64`
+    /** Duration of foundation pool inflation in years */
+    foundationTerm: number; // Until we land on best type for `f64`
+    /**
+     * Rate per year at which inflation is lowered.
+     * (Rate reduction is derived using the target
+     * slot time in genesis config)
+     */
+    taper: number; // Until we land on best type for `f64`
+    /** Terminal inflation percentage */
+    terminal: number; // Until we land on best type for `f64`
```

Comment:
Please do make a type for these like `F64Unsafe`, move the comment there, link to the GitHub issue, and maybe add a link to this: https://stackoverflow.com/a/57225494/802047